fvm/evm/stdlib/contract.cdc

```diff
@@ -442,10 +442,11 @@ contract EVM {
         }
 
         /// Withdraws the balance from the cadence owned account's balance
-        /// Note that amounts smaller than 10nF (10e-8) can't be withdrawn
+        /// Note that amounts smaller than 1e10 attoFlow can't be withdrawn
```

Comment:
```suggestion
        /// Note that amounts smaller than 1e10 attoFlow can't be withdrawn
        /// In other words, the smallest withdrawable balance is 1e10 attoFlow. 
```

Reply:
Updated in https://github.com/onflow/flow-go/pull/6877/commits/fb356030010e44ba020f716cebc4b89c7e497dc4 .